src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -93,19 +97,15 @@ class CombatLogParser extends MainCombatLogParser {
 
     const fightDuration = this.fightDuration;
 
-    const nonDpsTimePercentage = this.modules.alwaysBeCasting.totalDamagingTimeWasted / fightDuration;
     const deadTimePercentage = this.modules.alwaysBeCasting.totalTimeWasted / fightDuration;
 
-    console.log(this.modules.alwaysBeCasting.totalTimeWasted+'---'+fightDuration);
-
-    if (nonDpsTimePercentage > 0.01) {
-      results.addIssue({
-        issue: `Your non DPS time can be improved. Try to cast damaging spells more regularly.`,
-        stat: `${Math.round(nonDpsTimePercentage * 100)}% non DPS time (<30% is recommended)`,
-        icon: 'petbattle_health-down',
-        importance: getIssueImportance(nonDpsTimePercentage, 0.4, 0.45, true),
-      });
+    if(this.modules.abilityTracker.abilities[1] !== undefined)  {
```

Comment:
Please do not use any *magic numbers* in the code, either define them as a constant at the top of your file if they're only going to be used in one place and you don't need any additional info (e.g. `const MELEE_SPELL_ID = 1;`) or add a spell definition to the `SPELLS` file and use that (`SPELLS.MELEE.id`).

The same goes for the 4 abilities below.